samples/nrf9160/lwm2m_client/Kconfig

```diff
@@ -154,6 +154,9 @@ config APP_NEIGHBOUR_CELL_SCAN_INTERVAL
 	help
 	  Low value on interval may reduce the performance of the system.
 
+config APP_LWM2M_CONFORMANCE_TESTING
+	bool "Enable peridically Send operation after connection"
```

Comment:
```suggestion
	bool "Enable periodical SEND operation when connected"
```

Reply:
OK. Maybe I should change: Enabler for periodical SEND operation.  It was started after connection but Now it need Mute true and Mute false trigger for start.